support-models/src/main/scala/com/gu/support/workers/GiftRecipient.scala

```diff
@@ -70,41 +71,6 @@ object GeneratedGiftCode {
 
 }
 
-sealed trait GiftPurchase {
```

Comment:
this trait has been made redundant as it's all included in the new ProductTypeCreated type